mysql-test/include/audit_log_events.inc

```diff
@@ -37,6 +37,9 @@ alter table t1 rename renamed_t1;
 select * from t_doesnt_exist;
```

Comment:
True. The problem is, there is not "audit_log.test", but nothing stopped me from starting one.